gradio/layouts.py

```diff
@@ -113,12 +128,13 @@ class TabItem(BlockContext):
     components defined within the TabItem will be rendered within a tab.
     """
 
-    def __init__(self, label, **kwargs):
+    def __init__(self, label, id=False, **kwargs):
```

Comment:
Let's use `None` instead of `False` 

```suggestion
    def __init__(self, label:str, id:Optional[int | str] = None, **kwargs):
```

Reply:
Let's document these parameters